src/sections/Partners/partners.js

```diff
@@ -74,6 +81,7 @@ export const partners = [
+    partnerId: "istio",
```

Comment:
I think we don't need a partnerId if it is not connected from the home page

Reply:
I defined partnerId for all to same it even for all, as it is iterating through all partners so thought it might drop an error if I skip these IDs

Reply:
I don't think it will drop a error , so why add unimportant lines.

Reply:
ok , i will remove those ids 👍